pkgs/servers/dns/knot-dns/default.nix

```diff
@@ -26,11 +26,6 @@ stdenv.mkDerivation rec {
     # They are later created from NixOS itself.
     ./dont-create-run-time-dirs.patch
     ./runtime-deps.patch
-    # rename task_t to worker_task_t to fix redefinition issues on (aach64-)darwin
-    (fetchpatch {
```

Comment:
Please remove the unused fetchpatch from inputs.

Reply:
Oh right, thanks.